core/transformation/pathfinders/version_1.py

```diff
@@ -0,0 +1,491 @@
+    output_categories = outputs.map(OUTPUT_CATEGORIES)
+    melted_df = pd.melt(
+        df_dict["Outputs"],
+        id_vars=["Intervention theme", "Output", "Unit of measurement"],
+        var_name="Reporting Period",
+        value_name="Amount",
+    )
+    start_dates = melted_df["Reporting Period"].map(
+        lambda x: PF_REPORTING_PERIOD_TO_DATES[", ".join(x.split(", ")[:-1])]["start"]
+    )
+    end_dates = melted_df["Reporting Period"].map(
+        lambda x: PF_REPORTING_PERIOD_TO_DATES[", ".join(x.split(", ")[:-1])]["end"]
+    )
+    actual_forecast = melted_df["Reporting Period"].map(lambda x: "Actual" if "Actual" in x else "Forecast")
+    return {
+        "Outputs_Ref": pd.DataFrame(
+            {
+                "Output Name": outputs,
+                "Output Category": output_categories,
+            }
+        ),
+        "Output_Data": pd.DataFrame(
+            {
+                "Additional Information": pd.NA * len(melted_df),
+                "Project ID": [pd.NA] * len(melted_df),
+                "Output": melted_df["Output"],
+                "Unit of Measurement": melted_df["Unit of measurement"],
+                "Amount": melted_df["Amount"],
+                "Actual/Forecast": actual_forecast,
+                "Start_Date": start_dates,
+                "End_Date": end_dates,
+                "Programme ID": [programme_id] * len(melted_df),
+            }
+        ),
+    }
+
+
+def _outcomes(
+    df_dict: dict[str, pd.DataFrame],
+    programme_name_to_id_mapping: dict[str, str],
+) -> dict[str, pd.DataFrame]:
+    """
+    Populates `outcome_dim` and `outcome_data` tables:
+        For `outcome_dim`:
+            outcome_name     - from "Outcome" in the transformed DF "Outcome_Ref"
+            outcome_category - from "Outcome Category" in the transformed DF "Outcome_Ref"
+
+        For `outcome_data`:  # NOTE: This table schema is not finalised and may change
+            project_id              - from "Project ID" in the transformed DF "Outcome_Data"
+            programme_junction_id   - assigned during map_data_to_models based on "Programme ID" in the transformed DF
+                                      "Outcome_Data"
+            outcome_id              - assigned via FK relations in map_data_to_models based on "Outcome" in the
+                                      transformed DF "Outcome_Data"
+            start_date              - from "Start_Date" in the transformed DF "Outcome_Data"
+            end_date                - from "End_Date" in the transformed DF "Outcome_Data"
+            event_data_blob         - includes "Unit of Measurement", "Amount" and "Actual/Forecast" from the
+                                      transformed DF "Outcome_Data"
+    """
+    organisation_name = df_dict["Organisation Name"].iloc[0, 0]
+    programme_id = programme_name_to_id_mapping[organisation_name]
+    outcomes = df_dict["Outcomes"]["Outcome"]
+    outcome_categories = outcomes.map(OUTCOME_CATEGORIES)
```

Comment:
I don't think we'll be able to use these mappings because they don't include all the values from the PF template. We need to confirm mappings with PF as mentioned here https://dluhcdigital.atlassian.net/browse/SMD-655.

I think it will probably be best to store them in the template, that way we can still display the IT to the user without letting them select it. I think we should also mock out those mappings for now as well, instead of using the existing ones used for TF. 

Once confirmed and added to the spreadsheet we can also update the work done in https://github.com/communitiesuk/funding-service-design-post-award-data-store/pull/468 to include these mappings. What do you think?